### PR TITLE
Update Integrate with SendGrid Procedure 

### LINKED ISSUE #1231: Astro documentation for integrating email alert with sendgrid is incorrect
**Affected Document:**
https://docs.astronomer.io/astro/airflow-alerts#integrate-with-sendgrid

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
I have followed the steps to setup sendgrid, and found out the following:
image
In the step 6, the document mentions to add this backend, along with the airflow connection in step 7.
- This part never worked, and what I have found out is that the required package (https://pypi.org/project/apache-airflow-providers-sendgrid/) that contains the `airflow.providers.sendgrid.utils.emailer.send_email` is not included with the Astro runtime, thus this part never works.
- I was able to make it work by adding the environment variable mentioned in the following URL: https://airflow.apache.org/docs/apache-airflow/stable/howto/email-config.html#using-default-smtp as such:
image

**Additional Notes:**
- Also, in order for sendgrid to properly send out the email, you need to make sure the FROM EMAIL ID is identical to the send validated email that you set in sendgrid, due to the security reasons. The Astronomer documentation also does not seem to mention the importance of this as well.

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1231.

### CODE REVIEW

**Review by howardyoo — CHANGES_REQUESTED** [review]
Please review my comments and make appropriate additions!

File: astro/airflow-alerts.md [hunk 1]
```diff
@@ -42,14 +42,19 @@ The following topics provide setup steps for integrating each of these external
...
-6. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+6. Add the [SendGrid Airflow provider](https://airflow.apache.org/docs/apache-airflow-providers-sendgrid/stable/index.html) to your Astro project by adding the following to your `requirements.txt` file:
+
+    ```text
+    apache-airflow-providers-sendgrid
+    ```
+7. In the Deployment view of the Cloud UI, create an environment variable with the following values:
 
     - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
     - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
```
[the remark this answers is not in the document]
Comment (kwcanuck) [reply astro/airflow-alerts.md]: @howardyoo I have updated the procedure based on your feedback. Please review and provide your edits or approval.

File: astro/airflow-alerts.md [hunk 2]
```diff
@@ -47,12 +47,13 @@ The following topics provide setup steps for integrating each of these external
     ```text
     apache-airflow-providers-sendgrid
     ```
-7. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+7. In the Deployment view of the Cloud UI, add the following environment variables:
 
-    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
-    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
+    - `AIRFLOW__EMAIL__EMAIL_BACKEND` = `airflow.providers.sendgrid.utils`
```
Comment (howardyoo) [inline astro/airflow-alerts.md:52]: the value should be `airflow.providers.sendgrid.utils.emailer.send_email`, not `airflow.providers.sendgrid.utils`

**Review by howardyoo — CHANGES_REQUESTED** [review]
requesting small fix for the incorrect value.

File: astro/airflow-alerts.md (hunk 2 above)
Comment (kwcanuck) [reply astro/airflow-alerts.md:52]: @howardyoo I have updated the text as requested. In future, ff there are specific changes or additions you want:

1. Click **Files changed**.

2. Scroll to the location where you want to add or edit text and click the **Plus** (+) icon.

image

3. Click **Add a suggestion**.

image

4. Add or edit text and then click **Add single comment**.

Please review the changes and provide your edits, or approval.

Thanks!

Comment (howardyoo) [reply astro/airflow-alerts.md:52]: Oh ok, thanks! I got it.

File: astro/airflow-alerts.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (howardyoo) [reply astro/airflow-alerts.md]: Thanks! Looks good.

**Review by howardyoo — APPROVED** [review]
LGTM :-)

**Comment by kwcanuck** [comment]
@jwitz ready for your review.

File: astro/airflow-alerts.md [hunk 3]
```diff
@@ -42,14 +42,20 @@ The following topics provide setup steps for integrating each of these external
 
 5. Verify your integration in SendGrid to confirm that the key was activated. If you get an error indicating that SendGrid can't find the test email, try rerunning the cURL code in your terminal before retrying the verification.
 
-6. In the Deployment view of the Cloud UI, create an environment variable with the following values:
+6. Add the [SendGrid Airflow provider](https://airflow.apache.org/docs/apache-airflow-providers-sendgrid/stable/index.html) to your Astro project by adding the following to your `requirements.txt` file:
```
Comment (jwitz) [inline astro/airflow-alerts.md:45]:
```suggestion
6. Add the following line to your Astro project `requirements.txt` file to install the [SendGrid Airflow provider](https://airflow.apache.org/docs/apache-airflow-providers-sendgrid/stable/index.html):
```

File: astro/airflow-alerts.md [hunk 4]
```diff
@@ -42,14 +42,20 @@ The following topics provide setup steps for integrating each of these external
...
 
-    - **Key**: `AIRFLOW__EMAIL__EMAIL_BACKEND`
-    - **Value**: `airflow.providers.sendgrid.utils.emailer.send_email`
+    ```text
+    apache-airflow-providers-sendgrid
+    ```
```
Comment (jwitz) [inline astro/airflow-alerts.md:49]:
```suggestion
    ```

```

**Review by jwitz — APPROVED** [review]

**Pull request merged**